xml/System.Runtime.Intrinsics/Vector64.xml

```diff
@@ -67,6 +67,8 @@
           <paramref name="vector" /> reinterpreted as a <see cref="T:System.Runtime.Intrinsics.Vector64`1" /> of type <typeparamref name="U" />.</returns>
         <remarks>To be added.</remarks>
         <exception cref="T:System.NotSupportedException">The type of <paramref name="vector" /> (<typeparamref name="T" />) or the type of the target (<typeparamref name="U" />) is not supported.</exception>
+        <typeparam name="TFrom">The type of the input vector.</typeparam>
```

Comment:
This seems wrong. Were the type parameters renamed after .NET 6 GA?

Reply:
Yes, looking at the source it looks like they were renamed. I think we should redo this using the .NET 6 GA tag?

Reply:
I synced my runtime enlistment to the latest bits in main. I'll re-run the tool against this type for the runtime GA branch.

Reply:
I'll just revert the changes introduced in this file. This might be a bug in DocsPortingTool. It should check that the `<typeparam>` exists among the `<TypeParams>` items a few lines above.